### PR TITLE
Move house management function to Lua scripts

### PR DESCRIPTION
**Pull request by ranisalt**
This moves house management spells (aleta sio, aleta som, aleta grav and alana sio) to Lua scripts.

aleta sio and aleta som are working great, but I'm not sure about aleta grav. It inserts (and deletes, when erased) multiple rows at `house_lists` table, but ingame it works properly and it should not be related to these changes.

alana sio also works great but GM players can't kick themselves as they have the `CanEditHouses` flag set and [House::kickPlayer](https://github.com/otland/forgottenserver/blob/master/src/house.cpp#L160) disallows. I could fix while I'm at it.

### CODE REVIEW

**Comment by raymondtfr** [comment]
I am not sure but I think I saw @marksamman mentioning somewhere about moving **commands** and **house spells** to *talkactions*.

**Comment by Kamenuvol** [comment]
I believe they are spells because they are shown in spellbook. But even if he want to move to talkactions, it's easier to change now.

**Comment by ranisalt** [comment]
Currently, those 4 house commands are spells even in Tibia, I don't see the point in moving them to talkactions unless they change to be more like commands, e.g. `/guests` or `/kick playerName`. Internally spells are talkactions, too.

There is a PR (#1990) moving the remaining commands (`/raid`, `/reload` and `!sellhouse`) to talkactions.

File: data/spells/scripts/house/edit_door_list.lua [hunk 1]
```diff
@@ -0,0 +1,33 @@
+local function getNextPosition(player)
```
Comment (ninjalulz) [inline data/spells/scripts/house/edit_door_list.lua]: I think you should remove this redundant function, and use Position.getNextPosition instead.

Comment (ranisalt) [inline data/spells/scripts/house/edit_door_list.lua]: Sure! Didn't see it already existed in libs.

**Comment by ranisalt** [comment]
Let's discuss this spell vs talkaction later, open an issue if you want.

**Pull request merged**